localstack/services/awslambda/lambda_api.py

```diff
@@ -239,6 +241,14 @@ def build_mapping_obj(data) -> Dict:
     if data.get("DestinationConfig"):
         mapping["DestinationConfig"] = data.get("DestinationConfig")
 
+    if data.get("FilterCriteria"):
+        # validate for valid json
+        if not validate_filters(data.get("FilterCriteria")):
+            # AWS raises following Exception when FilterCriteria is not valid:
+            # An error occurred (InvalidParameterValueException) when calling the CreateEventSourceMapping operation:
+            # Invalid filter pattern definition.
+            raise InvalidParameterValueException("Invalid filter pattern definition.")
```

Comment:
Unfortunately using the `InvalidParameterValueException` will not be parsed to the expected error-response. We need to use `ValueError` like this:
```
raise ValueError(
                INVALID_PARAMETER_VALUE_EXCEPTION,"Invalid filter pattern definition.")
```